src/azure-cli/azure/cli/command_modules/appconfig/custom.py

```diff
@@ -34,7 +34,8 @@ def create_configstore(client,
                        location,
                        sku="Standard",
                        assign_identity=None,
-                       enable_public_network=None):
+                       enable_public_network=None,
+                       disable_local_auth=None):
```

Comment:
Could you also add some sample commands in the help file?

Reply:
Sample added.